### DISCUSSION TITLE
如何替换 Layout.Sider 的背景颜色？

### DISCUSSION THREAD

**Discussion by enzeberg** [ant-design/ant-design#39998] [Q&A]
官网 ( [https://ant.design/components/layout-cn#components-layout-demo-top-side-2](https://ant.design/components/layout-cn#components-layout-demo-top-side-2) ) 上改变 Sider 默认的背景颜色用的是 _行内样式_ ，我也没找到相应的可用于 `ConfigProvider` 的 token 。

我的代码的多个地方都用到了 Sider,  目前是一直使用 `style={{ backgroundColor: 'white', }}` ，既不优雅，也影响性能，所以想请问一下有没有在 `ConfigProvider` 中 **替换 Layout.Sider 背景颜色** 的方式？

**Comment by radiorz** [comment]
collapse 的时候还得修改 悬浮按钮的颜色...
